TIPS.md

```diff
@@ -6,26 +6,49 @@ Instructions for play
   - You can also press a number key to change the state (so press 2 to change the state to 2)
   - Note that invaders that have extra health or deal extra damage are rendered in a different color (i.e. red for england buildings, and blue for russian explorers)
 - Defend values can also be set as a state
 - To prevent a piece from going away during time passes you should lock it (right-click and select lock)
 - You can use energy tokens instead of the counter by deleting the energy counter (right-click on the energy counter and press delete)
+  - Note that deleting requires you to be the host or to be promoted
+- When removing blight from the blight card (usually for an event), move it to the "box" to the top-left of the play area (instead of deleting it)
+- When you have dealt enough damage to an invader to destroy it, drag it from the island board onto the "play mat" and add the corresponding amount of fear to the fear pool. 
+  - This has to be manual because some events change the amount of fear that you generate
   
 Advanced Tips:
 - UI elements can be enabled and disabled with the little triangle near the top-right of your screen
 - You can play double-handed (i.e. play 2 or more spirits) by changing your TTS player color in the top-right of the screen and choosing another spirit
   - Note that your UI elements are tracked per spirit
-- You can bind keys to many other actions with options->keys (TODO check name)
-- When playing as Shroud of Silent Mist it's important to not heal invaders and dahan in its land when time passes
-  - You can do this by locking the bottom invader in a stack or by stacking the damaged invader on top of a scenario token
+- You can bind keys to many other actions with Options (near the top-middle of the screen) -> Game Keys
 - If you play thresholded "Cast Down Into the Briny Deeps" use the gizmo tool (one of the tools on the left side of the screen) to move the island board
 - You can choose what island boards to play with at setup by placing the island boards before starting your game
 - To change what spirit you're playing after you've already started the game either reload the mod to restart everything, or unlock the spirit panel (right-click and press lock) and delete the spirit panel and any extra tokens/cards it has (right-click and select delete), then drag over a new spirit panel.
+- To play with an archipelago map, grab the map tiles from the box, which is to the left of the setup buttons, and then place them onto the map. When you hit Start Game after choosing your options the mod will set things up for you
+  - You probably want to hit the "Hide UI" button in the setup area to see more clearly where you are putting the boards
+  - This holds true for any custom map setup you want to do
+- To use save files posted in the Discord channel, download the JSON file and put it into your TTS saves folder: My Documents/My Games/Tabletop Simulator/Saves
```

Comment:
This isn't really needed to be in the notes in the mod, as it isn't a feature of the mod, and is more fit in the discord (where something similar is pinned)

Reply:
i think it's fine to still mention this here, so that the information is all compiled together. though the save files currently are only distributed on discord, which also includes instructions there, so i'd also be fine with removing this